scan.rb

```diff
@@ -11,13 +11,33 @@ def get_option(key)
   selected.split('=').last
 end
 
+def write_results(hostname, response, output) 
+  begin
+    puts " Start writing final results"
+    file = File.open("#{output}", "ab+")
```

Comment:
No need for string interpolation, just using `output` is enough.

Reply:
done